src/olympia/abuse/models.py

```diff
@@ -216,15 +233,6 @@ def report(cls, abuse_report):
             # we don't these additional requests to be retried, so reraise
             raise ConnectionError from exc
 
-        if cinder_job.decision and (
-            cinder_job.decision.action
-            == DECISION_ACTIONS.AMO_REJECT_VERSION_WARNING_ADDON
-        ):
-            # if this is a new report, it's never an appeal
-            cinder_job.decision.get_action_helper().notify_reporters(
-                reporter_abuse_reports=[abuse_report], is_appeal=False
-            )
```

Comment:
This was an obsolete chunk of code - it isn't possible for Cinder to return a job_id for a job that is already closed with a decision.